dense_vector/challenges/default.json

```diff
@@ -47,6 +52,36 @@
         "include-in-reporting": false
       }
     },
+    {
+      "parallel": {
+        "warmup-time-period": 10,
+        "completed-by": "index-update-concurrent-with-searches",
+        "tasks": [
+          {
+            "name": "index-update-concurrent-with-searches",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": 5000,
+              "ingest-percentage": 5,
+              "refresh" : "wait_for"
+            },
+            "clients": 1
+          },
+          {
+            "name": "knn-search-100-1000-concurrent-with-indexing",
+            "operation": "knn-search-100-1000",
+            "clients": 1
```

Comment:
this is the default, you can omit it

Reply:
Addressed in 5d96d31b6541bea50c7524cf58b094f2b3509eee